src/components/content/order/migrate/SelectDestination.tsx

```diff
@@ -79,9 +74,9 @@ export const SelectDestination = ({
     selectAvailabilityZones: Record<string, string>;
     setSelectAvailabilityZones: Dispatch<SetStateAction<Record<string, string>>>;
     currentFlavor: string;
-    billingModes: DeployRequest.billingMode[] | undefined;
-    selectBillingMode: string;
-    setSelectBillingMode: Dispatch<SetStateAction<string>>;
+    billingModes: Billing.defaultBillingMode[] | undefined;
```

Comment:
we must not use the defaultBillingMode type here.  Must be just the Billing.BillingMode